rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -148,7 +149,7 @@ def test_ppo_exploration_setup(self):
         obs = np.array(0)
 
         # TODO (Kourosh) Test against all frameworks.
-        for fw in framework_iterator(config, frameworks=("torch")):
+        for fw in framework_iterator(config, frameworks=("torch", "tf2")):
```

Comment:
oh actually in this case it doesn't matter because I'm using the framework iterator in kinda a dumb way here. I'm not using it to modify an algo config, I'm just using it to get a framework name.

setting eager_tracing to false only matters if we were modifying an algo config.